### DISCUSSION TITLE
Address not saved when back on checkout as guest

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#334] [1. Help]
Hi @vovayatsyuk 

If I start to checkout as a guest, enter my address, and go back on another page and come back again on checkout, the system forgets the address. it is not happening everytime but happening and if I direct hit URL of cart like checkout/cart it happens everytime, working fine with luma.

you can check here:
https://master-7rqtwti-vcte4ksdazidy.eu-4.magentosite.cloud/breeze/

Thanks,
Pankaj

**Comment by pk-pankaj** [comment]
@vovayatsyuk works fine when we exclude the cart page from the breeze, so seems it is resetting the checkout data when visiting cart page.

In default Magento "estimate-shipping-methods" run on the basket page also but when back directly from checkout but in case of breeze this request is not being triggered.

**Reply by vovayatsyuk** [reply]
Thanks for the details!

Fixed in this commit: https://github.com/breezefront/module-breeze/commit/9a37e9e5b3ca830814ddaf1fc8688acead9b89b7

**Comment by pk-pankaj** [comment]
Hi @vovayatsyuk 

We found another issue on the basket page with Breeze.

We have different tax rule on different countries. For say  UK has 20% and France has 0%.

After login , when we choose a different country on the checkout step ( for say France) while the default country is GB and go to the payment page and back to the home page or go to the basket page then subtotal is not getting update as per FR country.

We have hide the Estimated Shipping and Tax on sidebar so when we will show it, Country remain GB there while it must be France.

I checked in Luma and working well there.

And its working well Guest with Breeze but not in logged in case.

Thanks
Pankaj

**Reply by vovayatsyuk** [reply]
Can you elaborate, do you select one of the saved addresses, or enter a new one at checkout?

I'm asking because if I enter a new address, then everything works good in Luma and Breeze. But, if I select a previously saved address from the list, then the same issue happens on Luma and Breeze.

**Comment by pk-pankaj** [comment]
Hi @vovayatsyuk 

I have double-checked that this is a breeze issue and still exists after applying the patch above, it works fine when I exclude cart page from breeze.

Steps to replicate:

1. login customer
2. go to the checkout page with any address
3. then back on cart page 
4. go to again on the checkout page and select a different address which is already saved (do not add a new one on checkout), if you do not get an issue then repeat this process 2-3 times.

If you get the issue and want to check with Luma then do not forget to clear all the browser cache,  local storage, cookies etc, as it keep the info save and issue replicate in luma as well.

I have selected France on checkout
![image](https://github.com/user-attachments/assets/a5a4cdf5-77a3-4523-910c-25d62f08439c)
 
but showing UK on cart page when back

![image](https://github.com/user-attachments/assets/d99492b9-0d27-48fd-8a80-55e9bf6d0357)


This has been a critical issue for us so please check early if possible for you.

Thanks,
Pankaj
